UI/EmuScreen.cpp

```diff
@@ -1120,6 +1120,30 @@ static void DrawDebugStats(DrawBuffer *draw2d) {
 	draw2d->SetFontScale(1.0f, 1.0f);
 }
 
+static void DrawCrashDump(DrawBuffer *draw2d) {
+	char statbuf[4096];
+	char versionString[256];
+	sprintf(versionString, "%s", PPSSPP_GIT_VERSION);
+	// TODO: Draw a lot more information. Full register set, and so on.
```

Comment:
Or at some point we might even be able to break into the debugger. Though restoring the register state properly (some registers may be "cached") might be a bit iffy (or take a lot more metadata than we currently attach to blocks).

But yes, for now a warning seems appropriate.

Reply:
Yeah, potentially if this works well on all platforms, it might make sense to just have slow memory mean to flush before reads/writes, or even be automatic just when a debugger is attached.

-[Unknown]